ground-server/src/app/controls/page.tsx

```diff
@@ -144,6 +144,20 @@ export default async function Controls() {
             initialStateLabel="No Data"
             useSwitch={false}
           />
+
+          <ActuationBox
```

Comment:
We'll also need a slider for choosing how long to vent for - let's do 1, 2, 3, or 4 second options